tests/integration.js

```diff
@@ -111,6 +111,7 @@ function testCursor(browser) {
       e.pageX = 150;
       e.pageY = 200;
       Tools.board.dispatchEvent(e);
+      done();
```

Comment:
?

Reply:
In line 108 I was getting the below lint error:
'done' is declared but its value is never.
'done' is defined but never used. Allowed unused args must match /^_/u.
So, I have done some research and add done() in line 114.
Explaination:
In the context of an asynchronous function, the done function is often used as a callback to signal the completion of an asynchronous operation. It is a convention in testing frameworks, particularly in asynchronous testing, where you want to ensure that certain asynchronous tasks are completed before proceeding with the next step or completing the test.

You can check other functions in that file.

Reply:
the function is synchronous, you don't need the done function at all, do you ?

Reply:
I just have to add done() as you have add done in this line: `.execute(function (done) {`
```
function testCursor(browser) {
  return browser
    .execute(function (done) {
      Tools.setColor("#456123"); // Move the cursor over the board
      var e = new Event("mousemove");
      e.pageX = 150;
      e.pageY = 200;
      Tools.board.dispatchEvent(e);
      done();
    })
    .assert.cssProperty(
      "#cursor-me",
      "transform",
      "matrix(1, 0, 0, 1, 150, 200)",
    )
    .assert.attributeEquals("#cursor-me", "fill", "#456123");
}
```

if I remove done() from the line 114 then I have to remove it from line 108